### ISSUE TITLE
Please add Veteran to LMB Integration

### ISSUE THREAD

**Issue by kcaponeAP** [activeprospect/leadconduit-integration-lowermybills#15]
Currently the LMB integration does not support military status. I just received a recent XML from LMB with a new field addition.

Please have this map to the internal LCN field / mortgage.application.is_military

Options for this field are yes & no. 

```xml
...<REAgentPhone/>
<Veteran>yes</Veteran>  <------
<LeadCost>5</LeadCost>
<NeedSellingREAgent/>
```

**Comment by ros3bud** [comment]
In progress.

**Comment by kcaponeAP** [comment]
Do we have and update / ETA for this?

Ticket for reference: https://activeprospect.zendesk.com/agent/tickets/6631

**Comment by ros3bud** [comment]
Solved in Version 1.1.4
